app/services/submissions/create_manifest.rb

```diff
@@ -45,6 +45,7 @@ def write_manifest
         end
       end
       process_cc0_license unless submission.license.present?
+      process_on_behalf_of
```

Comment:
Consider adding `if submission.submitter.present?` to this line.  The way we're using this class now, it should always be there so, technically, adding it would be a 'YAGNI' violation at this point but it seems like a simple enough defensive move.

Reply:
Thanks @coblej, good idea!